coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
@mwharrison Did you mean to add the coverage files?

Reply:
I did not, these should be ignored. Let me see if I can remove them.